src/main/java/org/veupathdb/service/eda/us/model/UserDataFactory.java

```diff
@@ -376,6 +367,77 @@ public void transferGuestAnalysesOwnership(long fromGuestUserId, long toRegister
     );
   }
 
+  /***************************************************************************************
+   *** Read analysis metrics
+   **************************************************************************************/
+
+  public UserAnalysisMetricsResponse readAnalysisMetrics(Date startDate, Date endDate) {
+    UserAnalysisMetricsResponseImpl response = new UserAnalysisMetricsResponseImpl();
+    response.setStartDate(startDate);
+    response.setStartDate(endDate);
+    response.setCreationCounts(getAnalysisCountsPerDateType(startDate, endDate, "creation_time"));
```

Comment:
Maybe consider moving these table names to an enum name e.g. EventType { CREATION, MODIFICATION} with a getDbColumnName() method?